cli/test/README.md

```diff
@@ -695,6 +708,8 @@ This document includes CLI commands and the expected CLI output in order to help
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].tls`|**PENDING** (currently `TLS configuration not yet supported`)|N/A|
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].ttl`|**PENDING** (currently `TTL configuration not yet supported`)|N/A|
 |`fastddsqosprof file.xml set transport_descriptor[transport_id].wan_address`|**PENDING** (currently `WAN address configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].other`|`Transport <other> not recognized`|`TRANSPORT_USAGE`|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].other --help`|N/A|`TRANSPORT_USAGE`|
```

Comment:
This command is showing the following error `Transport <other> not recognized`. I also suggest to improve this message making it similar to others in the current CLI:

```
ERROR: Transport descriptor <other> element not recognized
```